pkg/scheduler/api/resource_info.go

```diff
@@ -437,38 +437,43 @@ func (r *Resource) Equal(rr *Resource, defaultValue DimensionDefaultValue) bool
 }
 
 // Diff calculate the difference between two resource object
-func (r *Resource) Diff(rr *Resource) (*Resource, *Resource) {
+// Note: if `defaultValue` equals `Infinity`, the difference between two values will be `Infinity`, marked as -1
+func (r *Resource) Diff(rr *Resource, defaultValue DimensionDefaultValue) (*Resource, *Resource) {
+	leftRes := r.Clone()
```

Comment:
The Clone() function added recently causes 400ms+ latency in scheduling period while submit 1w pod.  We'd better think out a way without using Clone().

Reply:
> The Clone() function added recently causes 400ms+ latency in scheduling period while submit 1w pod. We'd better think out a way without using Clone().

OK. I think it's better to open another issue to optimize performance totally. As to this fix, the input parameter is a pointer and we should not make any modifications at the original object. So clone is needed.

Reply:
That's ok to me to open another issue to improve the performance.